pkg/authentication/handlers.go

```diff
@@ -87,6 +88,10 @@ func (a *API) handleLogin(w http.ResponseWriter, r *http.Request) {
 	// add the Otel HTTP Client
 	r = r.WithContext(OtelHTTPClientContext(r.Context()))
 
+	if nextTo := r.URL.Query().Get("next"); nextTo != "" {
+		a.cookieManager.SetNextToCookie(w, nextTo)
+	}
```

Comment:
We need to validate the value of `next`. We should allow redirects only to our domain, otherwise anyone will be able to get the session cookies